### PR TITLE
DTSBPS-686: adding fortify scan

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
DTSBPS-686

### Change description ###
adding fortify scan

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

**Review by jasonpaige — APPROVED** [review]

File: config/fortify-client.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=65430
```
Comment (rajaram-nagarajan) [inline config/fortify-client.properties:1]: You need to use the release id of send-letter-service here. ID should be 65508, but better to double check.

**Review by rajaram-nagarajan — CHANGES_REQUESTED** [review]

**Review by rajaram-nagarajan — CHANGES_REQUESTED** [review]
Also, change your branch name to start with 'nightly' (ex: nightly-fod-scan-dtsbps-686) to easily understand the task by its name in jenkins.

File: config/fortify-client.properties (hunk 1 above)
Comment (rishikrsharma) [reply config/fortify-client.properties]: I have updated the SLS release id

**Comment by rishikrsharma** [comment]
> nightly-fod-scan-dtsbps-686

sure

**Pull request closed without merging**